code/include/rlbox_sandbox.hpp

```diff
@@ -332,16 +364,25 @@ class rlbox_sandbox : protected T_Sbx
     this->impl_free_in_sandbox(ptr.get_raw_sandbox_value());
   }
 
+  /**
+   * @brief Check if two pointers are in the same sandbox.
+   */
   static inline bool is_in_same_sandbox(const void* p1, const void* p2)
   {
     return T_Sbx::impl_is_in_same_sandbox(p1, p2);
   }
 
+  /**
+   * @brief Check if the pointer points to this sandbox's memory.
+   */
   inline bool is_pointer_in_sandbox_memory(const void* p)
   {
     return this->impl_is_pointer_in_sandbox_memory(p);
   }
 
+  /**
+   * @brief Check if the pointer points to application memory.
+   */
   inline bool is_pointer_in_app_memory(const void* p)
   {
     return this->impl_is_pointer_in_app_memory(p);
```

Comment:
For the null sandbox both `is_pointer_in_sandbox_memory` and `is_pointer_in_app_memory` return for any pointer as all pointers are both in the sandbox and app memory. For the wasm sandbox, these are mutually exclusive.